pjmedia/src/pjmedia/wav_player.c

```diff
@@ -416,6 +416,21 @@ PJ_DEF(pj_status_t) pjmedia_wav_player_port_create( pj_pool_t *pool_,
 
     /* It seems like we have a valid WAVE file. */
 
+    /* Check compatibility of sample rate and ptime.
+    * Some combinations result in a fractional number of samples per frame
+    * which we do not support.
+    * One such case would be for example 10ms @ 22050Hz which would yield
+    * 220.5 samples per frame.
+    */
+    if (0 != (ptime * wave_hdr.fmt_hdr.sample_rate *
+              wave_hdr.fmt_hdr.nchan % 1000)) {
```

Comment:
Minor: coding style, brace has its own line for multiple lines statements/conditions.